src/js/app.js

```diff
@@ -0,0 +1,65 @@
+$('document').ready(function() {
```

Comment:
You'll want to pass in just `document`, because you want to make sure the page is ready. Not a string 😄